### DISCUSSION TITLE
Sentry error reporting supports only old Falcon 2.0 :-(

### DISCUSSION THREAD

**Discussion by liborjelinek** [falconry/falcon#2163] [General]
As stated in the [Sentry Falcon integration docs](https://docs.sentry.io/platforms/python/guides/falcon/) it supports only Falcon 1.4 - 2.0...

How to report errors to Sentry from current 3.x Falcon? Or, what alternatives do you use for exception reporting?

**Comment by vytas7** [comment]
It looks like Sentry _should_ support 3.x by now as https://github.com/getsentry/sentry-python/pull/1297 was merged last year. I haven't verified this myself though.

Could you maybe give it a try and see how it goes?

As said, we will otherwise first focus on improving our story wrt the OpenTelemetry protocol, as it seems to be gaining traction among different providers and monitoring systems.

**Reply by liborjelinek** [reply]
Thank you. I confirm that Sentry works with Falcon 3.x perfectly.
